packages/functionals/botpress-nlu/src/index.js

```diff
@@ -218,9 +218,14 @@ module.exports = {
 
     router.get('/sync', async (req, res) => {
       try {
+        bp.events.emit('toast.nlu-sync', { text: 'NLU In Progress', type: 'info', time: 120000 })
```

Comment:
@dmytropanontko , in terms of UX, not sure we should only display toast for 120 seconds. Also not sure we should hide first toast if second one appears.

Reply:
I think _info_ toasts should auto-hide (so I agree with custom implementation), but _error_ or _warning_ toasts should persist till the user action

Reply:
sounds good, thanks